tests/juniper/juniper_base_protocol_test.py

```diff
@@ -26,7 +26,7 @@
 from tests.util.global_reactor import juniper_switch_ip, juniper_switch_netconf_port
 
 
-class JuniperBaseProtocolTest(unittest.TestCase):
+class JuniperBaseProtocolTest(BaseJuniper):
     def setUp(self):
         self.nc = self.create_client()
```

Comment:
This should be moved to BaseJuniper,
It uses it and depends on the children creating this variable.